pkg/packagekit/updates.scss

```diff
@@ -153,6 +153,20 @@ div.changelog {
   }
 }
 
+/* Update error view: same styles as of updating view */
+.update-error-main-view {
+  max-width: 60rem;
+  margin: 0 auto;
+}
+
+.error-log {
+  text-align: left;
+
+  .error-log-content {
+    margin: 0 10ex;
+  }
+}
+
```

Comment:
It's generally best to not add custom CSS unless it's needed (as it _can_ be brittle and would need to be supported, plus PF usually gives us ways to do things without adding custom CSS). In this case, I we could use PF widgets in the code itself and (probably) skip the CSS.

Some stuff, like alignment, also has PF utility classes now as well. https://www.patternfly.org/v4/utilities/alignment

Utility classes require 1 import on the top-level CSS (near the top of the file) for the page, if they're not already added in that file. (Do _not_ add them to imported files, else that causes a very high risk of CSS duplication.)

I think it'd be imported like this:

```scss
@import "@patternfly/patternfly/utilities/Alignment/alignment.css";
```